common/src/main/java/org/apache/sedona/common/geography/Constructors.java

```diff
@@ -67,4 +68,174 @@ public static Geography geogCollFromText(String wkt, int srid) throws ParseExcep
     }
     return geogFromWKT(wkt, srid);
   }
+
+  public static Geometry geogToGeometry(Geography geography, GeometryFactory geometryFactory)
+      throws ParseException {
+    return geogToGeometry(geography, geometryFactory, geography.getSRID());
+  }
+
+  public static Geometry geogToGeometry(
+      Geography geography, GeometryFactory geometryFactory, int srid) {
+    if (geography == null) return null;
```

Comment:
The `srid` is not used in `geogToGeometry(Geography geography, GeometryFactory geometryFactory, int srid)`.

A better approach is to define these 2 functions:

```
Geometry geogToGeometry(Geography geography, GeometryFactory geometryFactory); #1
Geometry geogToGeometry(Geography geography, int srid); #2
```

`GeometryFactory` object has an SRID property, the geometry objects created by that factory will automatically have that SRID. The variant #1 will implicitly use the SRID of the `geometryFactory`.

Variant #2 creates a new `GeometryFactory` object with specified `srid`, and calls #1 to create geometry objects with specified SRID.